libs/core/src/lib/panel/panel.component.ts

```diff
@@ -74,7 +74,10 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit {
     panelContent: PanelContentDirective;
 
     /** @hidden */
-    constructor(private _cdRef: ChangeDetectorRef, private _elementRef: ElementRef) {}
+    _rtl$ = this._rtlService.rtl;
+
+    /** @hidden */
+    constructor(private _cdRef: ChangeDetectorRef, private _elementRef: ElementRef, private _rtlService: RtlService) {}
```

Comment:
RtlService should have Optional decorator

Reply:
@JKMarkowski fixed